### PR TITLE
Introduce URLPopUp section.

### PR DESCRIPTION
**Pull request by gokaysatir**
Change-Id: I2cd85679c6de65fa7f999907a818d7406402b581

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by gokaysatir** [comment]
@pedropintosilva hello, can you have a look at the design of the hyperlink popup when you have time?

File: browser/src/canvas/sections/URLPopUpSection.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+		if (app.map['wopi'].EnableRemoteLinkPicker)
+			app.map.fire('postMessage', { msgId: 'Action_GetLinkPreview', args: { url: url } });
+    }
+
+	createUIElements(url: string) {
+		const parent = this.getHTMLObject();
+		L.DomUtil.createWithId('div', this.containerId, parent);
+
+        var link = L.DomUtil.createWithId('a', this.linkId, parent);
```
Comment (eszkadev) [inline browser/src/canvas/sections/URLPopUpSection.ts:38]: formatting -> `make prettier-write`

Comment (eszkadev) [reply browser/src/canvas/sections/URLPopUpSection.ts:38]: var -> let or const

File: browser/src/canvas/sections/URLPopUpSection.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+		const imgRemoveBtn = L.DomUtil.create('img', 'hyperlink-pop-up-removeimg', removeBtn);
+		L.LOUtil.setImage(imgRemoveBtn, 'lc_removehyperlink.svg', app.map);
+		imgRemoveBtn.setAttribute('width', 18);
+		imgRemoveBtn.setAttribute('height', 18);
+		imgRemoveBtn.setAttribute('style', 'padding: 4px');
+	}
+
+	setUpCallbacks() {
+		document.getElementById(this.linkId).onclick = function() {
```
Comment (eszkadev) [inline browser/src/canvas/sections/URLPopUpSection.ts:73]: please use arrow functions in the new code, then `this` should be already the right thing

**Comment by eszkadev** [comment]
It has very retro style now :)
linkpopup

Also no padding.

**Comment by gokaysatir** [comment]
@eszkadev thanks for checking :) Can you have a second look when you have time?

**Comment by pedropintosilva** [comment]
Additionally the position: to the x coordinate we should subtract the width of the section -> so, the popup appears horizontally centered in relation to the mouse cursor

**Comment by eszkadev** [comment]
copy link button doesn't work - but it seems to be unrelated to the patch

**Comment by eszkadev** [comment]
text is not vertically aligned with buttons in the popup

**Review by eszkadev — APPROVED** [review]
code looks ok

**Comment by gokaysatir** [comment]
> text is not vertically aligned with buttons in the popup

No it's not. We have a tracker for that issue.

**Pull request merged**